static/css/app.css

```diff
@@ -202,16 +217,16 @@ ol.progtrckr li.progtrckr-running.error::before {
 ol.progtrckr li.progtrckr-running.error {
     border-color: red;
 }
-
+/*
```

Comment:
Why both comment this out and change something?

If it's commented out, I'd rather see it removed out of the css :+1: